moveit_ros/move_group/src/move_group.cpp

```diff
@@ -290,8 +290,26 @@ int main(int argc, char** argv)
   const auto moveit_cpp = std::make_shared<moveit_cpp::MoveItCpp>(nh, moveit_cpp_options);
   const auto planning_scene_monitor = moveit_cpp->getPlanningSceneMonitorNonConst();
 
-  if (planning_scene_monitor->getPlanningScene())
+  if (auto ps = planning_scene_monitor->getPlanningScene())
   {
+    if (nh->has_parameter("default_planning_scene"))
```

Comment:
I am a bit concerned about tying this to only the existence of the parameter. How about checking whether or not the parameter is set.
```cpp
const auto default_planning_scene_file = std::string();
nh->get_parameter_or("default_planning_scene", default_planning_scene_file, std::string());
// Maybe add function to validate the string pattern if it actually representing  path
if(validateFilePath(default_planning_scene_file))
{
...
}


```